packages/common/src/feedback/fetch.ts

```diff
@@ -0,0 +1,69 @@
+import { IItem, getItem } from "@esri/arcgis-rest-portal";
+import {
+  IHubFeedback,
+  IHubRequestOptions,
+  IModel,
+  fetchModelFromItem,
+  fetchSetting,
+  getDefaultEntitySettings,
+  getItemBySlug,
+  isGuid,
+} from "..";
```

Comment:
Ensure you import from files - importing from folders or relative paths seems to cause gnarly problems with the actual exports